vim-plugins-profile.rb

```diff
@@ -1,14 +1,19 @@
 #!/usr/bin/env ruby
 # encoding: utf-8
 
+# Users can pass "nvim" as a first argument to use neovim.
+vim = ARGV.shift
+vim ||= "vim"
```

Comment:
I think `vim = ARGV.first || 'vim'` is cleaner. But that's only a tiny improvement. It's absolutely mergeable.